lighthouse/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM node:12-alpine
+FROM node:18-alpine
 
 RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories \
     && apk add chromium
 
 RUN npm install -g @lhci/cli@0.6.x
```

Comment:
do we want to look at updating `@lhci/cli`? That version is 3 years old...  https://www.npmjs.com/package/@lhci/cli/v/0.6.1

Reply:
Defo. I'll take a look now

Reply:
Updated the version to `0.12.x` and ran locally. Seems to be fine.